virtualizarr/manifests/group.py

```diff
@@ -0,0 +1,36 @@
+from typing import TypeAlias
+
+from zarr.core.group import GroupMetadata
+
+from virtualizarr.manifests import ManifestArray
+
+ManifestDict: TypeAlias = dict[str, ManifestArray]
```

Comment:
Rename to something that includes the word array? This makes it sound like it only contains the manifest itself